Assets/LeapMotion/Scripts/Utils/Utils.cs

```diff
@@ -5,12 +5,23 @@
 \******************************************************************************/
 
 using UnityEngine;
-using System.Collections;
-using Leap;
 
 namespace Leap.Unity {
 
   public static class Utils {
+    
+    /// <summary>
+    /// Similar to Unity's Transform.LookAt(), but resolves the forward vector of this
+    /// Transform to point away from the argument Transform.
+    /// 
+    /// Useful for billboarding Quads and UI elements whose forward vectors should match
+    /// rather than oppose the Main Camera.
+    /// </summary>
+    public static void LookAwayFrom(this Transform thisTransform, Transform transform) {
```

Comment:
No you can't, Vector3.up is not a compile-time constant, nor is new Vector3(0F, 1F, 0F)